test/e2e/README.md

```diff
@@ -0,0 +1,93 @@
+# ORAS End-to-End Testing Dev Guide
+
+## Setting up
+Minimal setup: Run the script in **step 3**
+
+### 1. Common dev setup for ORAS CLI
+https://hackmd.io/_nRHGW8WRfOOvngWc6u-sQ
```

Comment:
Please check-in this document in this PR.

Reply:
Since the original doc invloves vscode-related setup steps, will rewrite this step